pkg/controller/kubelet-config/kubelet_config_bootstrap.go

```diff
@@ -43,6 +43,7 @@ func RunKubeletBootstrap(templateDir string, kubeletConfigs []*mcfgv1.KubeletCon
 			if err != nil {
 				return nil, err
 			}
+
```

Comment:
Is it my view or is the only change in this file a blank line now (there used to be code here)? If just a blank line, let's remove that. :)

Reply:
Addressed